app/controllers/api/cloud_networks_controller.rb

```diff
@@ -9,5 +9,22 @@ def delete_resource(type, resource_id, _data = {})
       task_id = cloud_network.delete_cloud_network_queue(User.current_user.userid)
       action_result(true, "Deleting #{cloud_network.name}", :task_id => task_id)
     end
+
+    def options
+      return super unless params.keys.include_any?(%w[id ems_id])
+
+      if params.key?("ems_id")
+        ems = ExtManagementSystem.find(params["ems_id"])
```

Comment:
Should use the `resource_search` method here: `resource_search(params["ems_id"], :ext_management_systems, ExtManagementSystem)`

Reply:
`ems = resource_search(params["ems_id"], :ems, ExtManagementSystem)`

Is this correct?